nodes/node_balancer/cmd/clients.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"log"
+	"reflect"
+	"time"
+
+	configs "github.com/bugout-dev/moonstream/nodes/node_balancer/configs"
+)
+
+var clientPool ClientPool
+
+// Add client node and client itself if doesn't exist
+// TODO(kompotkot): Add mutes as for balancer
+func (cpool *ClientPool) AddClientNode(ip, blockchain string, node *Node) {
```

Comment:
instead of `ip`, use `clientID`. It will allow us to generalize in the future.

Reply:
Done, it reades client header